DANS-KNAW/easy-ingest-flow
# daemon init fix

fixes EASY-1138

#### When applied it will
* make a clearer separation between the REST and Hazelcast variants of the execution, such that `easy-ingest-dispatcher` does not need to start a Hazelcast client whenever the system runs in `rest` mode rather than `hazelcast` mode

#### Where should the reviewer @DANS-KNAW/easy start?
Start at `EasyIngestFlow`, as well as the dependent pull request in EasyIngestDispatcher: https://github.com/DANS-KNAW/easy-ingest-dispatcher/pull/34

#### How should this be manually tested?
* Deploy in `hazelcast` mode and check that it requires `easy-hazelcast` to be running
* Deploy in `rest` mode and check that it does not require `easy-hazelcast` to be running nor that it makes any connection with `easy-hazelcast`, even if it is running

#### related pull requests on github
repo                       | PR
-------------------------- | -----------------
easy-ingest-dispatcher                      | [PR#34](https://github.com/DANS-KNAW/easy-ingest-dispatcher/pull/34) 
easy-dtap | [PR#47](https://github.com/DANS-KNAW/easy-dtap/pull/47)
easy-hazelcast | [PR#3](https://github.com/DANS-KNAW/easy-hazelcast/pull/3)
easy-pid-generator | [PR#12](https://github.com/DANS-KNAW/easy-pid-generator/pull/12)
